Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -2047,9 +2047,9 @@ def convert_build_to_os_name(self, build):
             return 'Unknown'
 
         build_to_name_mapping = {
+            '26': 'Tahoe',
             '15': 'Sequoia',
-            '14': 'Sonoma',
-            '13': 'Ventura',
+            '14': 'Sonoma'
```

Comment:
Let's keep the trailing comma, which makes the diff here more obvious:

```suggestion
            '14': 'Sonoma',
```